internal/cephes/unity.go

```diff
@@ -0,0 +1,161 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+ * Cephes Math Library Release 2.4:  March,1996
+ * Copyright 1984, 1996 by Stephen L. Moshier
+ */
+
+/*
+ * Adapted from scipy's cephes unity.c
+ */
+
+package cephes
+
+import "math"
+
+const (
+	sqrt2    float64 = 1.414213562373095048801688724209698079 // sqrt(2)
```

Comment:
Some of these can be replaced with references to the `math` library, for example `pi4 = math.Pi / 4`